data_diff/diff_tables.py

```diff
@@ -381,6 +381,13 @@ def _bisect_and_diff_tables(self, table1, table2, level=0, max_rows=None):
         if max_rows < self.bisection_threshold:
             rows1, rows2 = self._threaded_call("get_values", [table1, table2])
             diff = list(diff_sets(rows1, rows2))
+
+            # This happens when the initial bisection threshold is larger than
+            # the table itself.
+            if level == 0 and not self.stats.get("table1_count", False):
```

Comment:
Good catch. But I think we always want to override these attribute. Why would we care what previous values they held?

Reply:
(talked about on slack, just for the record) Will double count if it's not level 0, but we can remove the not